src/qt/stattext.cpp

```diff
@@ -35,6 +35,11 @@ wxStaticText::wxStaticText(wxWindow *parent,
     Create( parent, id, label, pos, size, style, name );
 }
 
+wxStaticText::~wxStaticText()
+{
+    GetQLabel()->setBuddy( nullptr );
```

Comment:
I'm not familiar with wxQt details, but it might be worth adding a comment explaining why do we need to do this here?